packages/ui/src/styles/globals.css

```
@@ -2,6 +2,15 @@
 @import "./elevation.css";
 @source "../components/**/*.{ts,tsx}";
 
+@theme {
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)